src/formatDate.js

```diff
@@ -50,7 +50,59 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newFormat = [];
+  const newSeparator = toFormat[3];
+  const oldSeparator = fromFormat[3];
+  const oldDate = date.split(oldSeparator);
+  let YY;
+  let YYYY;
+  let MM;
+  let DD;
+
+  for (let i = 0; i <= fromFormat.length - 2; i++) {
+    if (fromFormat[i].includes('YY')) {
+      if (fromFormat[i].length > 2) {
```

Comment:
Let's check explicitly the string `YY` or `YY`

Reply:
done